### PR TITLE
Adding Data to backdrop.json

### PR DESCRIPTION
**Pull request by SillyInventor**
Backdrops.json

### Offset Backdrop Images

Useful to solving [this issue.](https://github.com/LLK/scratch-gui/issues/36)  

Adds the correct sizes of images in the backdrops library

### Reason for Changes

Because it allows you to display backdrops for a playtest

### Test Coverage

I didn't add new testing

### CODE REVIEW

File: src/lib/libraries/backdrops.json [hunk 1]
```diff
@@ -8,7 +8,11 @@
             "city",
             "sports"
         ],
-        "info": []
+        "info": [
+            480,
+            360,
+            1
+            ]
```
Comment (thisandagain) [inline src/lib/libraries/backdrops.json]: Inconsistent indentation.

**Comment by SillyInventor** [comment]
I made the mistake of not making a separate branch for the import changes. Needs new approval... also I need to change the checks.

**Comment by thisandagain** [comment]
@SillyInventor Thanks. Could you please rollback the changes to this initial PR and open a new PR for the import changes? These need to be reviewed separately and by different people.

/cc @rschamp

**Comment by SillyInventor** [comment]
Changes have been removed.

**Pull request merged**